hacktoberfest/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+    <title>Hacktoberfest</title>
+  </head>
+  <body>
+    <header>
+      <a href="https://discord.com/invite/hacktoberfest">Hacktoberfest Event Discord</a>
+    </header>
+    <section>
+      <h1>
+        How to Participate?
+      </h1>
+      <ol>
+        <li>
+          Create a Github account
+        </li>
+        <li>
+          <a href="https://hacktoberfest.com/register/">Register on the Hacktoberfest site</a>
+        </li>
+        <li>
+          Find a project
+        </li>
+        <li>
+          Fork the project
+        </li>
+        <li>
+          Clone to your computer
+        </li>
+        <li>
+          Add your code
+        </li>
+        <li>
+          Commit
+        </li>
+        <li>
+          Push
+        </li>
+        <li>
+          Do a Pull Request
+        </li>
+      </ol>
+    </section>
+    <section>
+      <h2>
+        Good Beginner Projects
+      </h2>
+      <ul>
+        <li>
+          <a href="https://goodfirstissue.dev/">https://goodfirstissue.dev/</a>
+        </li>
+        <li>
+          <a href="https://www.codetriage.com/">https://www.codetriage.com/</a>
+        </li>
+        <li>
+          <a href="https://www.firsttimersonly.com/">https://www.firsttimersonly.com/</a>
+        </li>
+        <li>
+          <a href="https://up-for-grabs.net/">https://up-for-grabs.net/</a>
+        </li>
+        <li>
+          <a href="https://codetribute.mozilla.org/">https://codetribute.mozilla.org/</a>
+        </li>
+      </ul>
+    </section>
+  </body>
+</html>
```

Comment:
Files should end with an empty return to help Git with EOL.
```suggestion
</html>

```